src/Helpers/SerializationHelper.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sammyjo20\LaravelHaystack\Helpers;
+
+use Illuminate\Support\Str;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Database\PostgresConnection;
+
+class SerializationHelper
+{
+    /**
+     * Serialize the given value.
+     */
+    public static function serialize(mixed $value): string
+    {
+        $serialized = serialize($value);
+
+        return self::isPgsql()
+            ? base64_encode($serialized)
+            : $serialized;
```

Comment:
This doesn't need to be put on two lines

```suggestion
        return self::isPgsql() ? base64_encode($serialized) : $serialized;
```

Reply:
Fixed.